libraries/AP_Motors/AP_MotorsHeli_Dual.h

```diff
@@ -87,8 +87,8 @@ class AP_MotorsHeli_Dual : public AP_MotorsHeli {
     const char* _get_frame_string() const override { return "HELI_DUAL"; }
 
     //  objects we depend upon
-    AP_MotorsHeli_Swash        _swashplate1;        // swashplate1
-    AP_MotorsHeli_Swash        _swashplate2;        // swashplate2
+    AP_MotorsHeli_Swash _swashplate1 { CH_1, CH_2, CH_3, CH_7 }; // swashplate1
+    AP_MotorsHeli_Swash _swashplate2 { CH_4, CH_5, CH_6, CH_8 }; // swashplate2
```

Comment:
Not a now thing, but I think a minor future improvement would be reorder the motor config in heli so the swashplate servo assignment of the first swash matches a single heli, just to make setup a little more consistent for the user.